schema.json

```diff
@@ -0,0 +1,496 @@
+                                    "type": "object",
+                                    "additionalProperties": false,
+                                    "maxProperties": 2,
+                                    "properties": {
+                                        "icon.svg": {
+                                            "type": "string"
+                                        },
+                                        "icon.png": {
+                                            "type": "string"
+                                        }
+                                    }
+                                },
+                                "host_os_tags": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string"
+                                    }
+                                },
+                                "project_type_tags": {
+                                    "type": "array",
+                                    "uniqueItems": true,
+                                    "items": {
+                                        "type": "string",
+                                        "enum": [
+                                            "ios",
+                                            "macos",
+                                            "android",
+                                            "xamarin",
+                                            "react-native",
+                                            "cordova",
+                                            "ionic",
+                                            "flutter",
+                                            "web"
+                                        ]
+                                    }
+                                },
+                                "type_tags": {
+                                    "type": "array",
+                                    "minItems": 1,
+                                    "maxItems": 1,
+                                    "items": {
+                                        "type": "string",
+                                        "enum": [
+                                            "access-control",
+                                            "artifact-info",
+                                            "installer",
+                                            "deploy",
+                                            "utility",
+                                            "dependency",
+                                            "code-sign",
+                                            "build",
+                                            "test",
+                                            "notification"
+                                        ]
+                                    }
+                                },
+                                "dependencies": {
```

Comment:
this is deprecated by now, isn't it?

Reply:
it is deprecated, but in older step versions this field still exists, so it has a format check in the schema